Kudu.Core/packages.config

```diff
@@ -18,6 +33,22 @@
   <package id="NuGet.Protocol.Types" version="3.0.0-pre-20150220053306" targetFramework="net45" />
   <package id="NuGet.Protocol.V2V3" version="3.0.0-pre-20150220053309" targetFramework="net45" />
   <package id="NuGet.Versioning" version="1.0.7" targetFramework="net45" />
+  <package id="System.Collections" version="4.0.11" targetFramework="net46" />
+  <package id="System.Collections.Concurrent" version="4.0.12" targetFramework="net46" />
+  <package id="System.ComponentModel" version="4.0.1" targetFramework="net46" />
+  <package id="System.Diagnostics.Debug" version="4.0.11" targetFramework="net46" />
+  <package id="System.Globalization" version="4.0.11" targetFramework="net46" />
+  <package id="System.IO" version="4.1.0" targetFramework="net46" />
   <package id="System.IO.Abstractions" version="1.4.0.74" targetFramework="net45" />
+  <package id="System.Linq" version="4.1.0" targetFramework="net46" />
+  <package id="System.Linq.Expressions" version="4.1.0" targetFramework="net46" />
+  <package id="System.Reflection" version="4.1.0" targetFramework="net46" />
+  <package id="System.Resources.ResourceManager" version="4.0.1" targetFramework="net46" />
+  <package id="System.Runtime" version="4.1.0" targetFramework="net46" />
+  <package id="System.Runtime.Extensions" version="4.1.0" targetFramework="net46" />
+  <package id="System.Runtime.InteropServices" version="4.1.0" targetFramework="net46" />
+  <package id="System.Text.Encodings.Web" version="4.0.0" targetFramework="net46" />
+  <package id="System.Threading" version="4.0.11" targetFramework="net46" />
+  <package id="System.Threading.Tasks" version="4.0.11" targetFramework="net46" />
```

Comment:
I tested against kudu verison 58.50929.2438

the cold start function key retrieval is respectively **7000ms : 11500ms,**
this change will adds roughly 4 sec

also the following assemblies were not loaded, I guess some other functionalities of `dataprotection` depends are them, but they are not being used here

```
 +  <package id="System.Collections.Concurrent" version="4.0.12" targetFramework="net46"/>
 +  <package id="System.Collections" version="4.0.11" targetFramework="net46" />
 +  <package id="System.ComponentModel" version="4.0.1" targetFramework="net46" />
 +  <package id="System.Diagnostics.Debug" version="4.0.11" targetFramework="net46" />
 +  <package id="System.Globalization" version="4.0.11" targetFramework="net46" />
 +  <package id="System.IO" version="4.1.0" targetFramework="net46" />
 +  <package id="System.Linq" version="4.1.0" targetFramework="net46" />
 +  <package id="System.Linq.Expressions" version="4.1.0" targetFramework="net46" />
 +  <package id="System.Reflection" version="4.1.0" targetFramework="net46" />
 +  <package id="System.Runtime.Extensions" version="4.1.0" targetFramework="net46" />      
 +  <package id="System.Threading" version="4.0.11" targetFramework="net46" />
 +  <package id="System.Threading.Tasks" version="4.0.11" targetFramework="net46" />
```